src/pot.jl

```diff
@@ -154,4 +154,19 @@ function sinkhorn_unbalanced2(
     )[1]
 end
 
+function smooth_ot_dual(
+    mu, nu, C, eps; reg_type="l2", method="L-BFGS-B", tol=1e-9, max_iter=500, verbose=false
+)
+    return pot.smooth.smooth_ot_dual(
+        nu,
+        mu,
```

Comment:
Just one to mention that I changed the Python wrappers in POT.jl such that it is not necessary to pass the arguments in the different order and all POT keyword arguments are supported and set to their Python defaults.

Reply:
Got it, sounds good. Once POT.jl is available from Pkg I can create a pull request with updated tests.